config.template.yml

```diff
@@ -553,4 +553,62 @@ notifier:
   #   sender: admin@example.com
   #   host: smtp.gmail.com
   #   port: 587
+
+##
+## Identity Providers
+##
+identity_providers:
+
+  ##
+  ## OpenID Connect (Identity Provider)
+  ##
+  ## It's recommended you read the documentation before configuraiton of this section:
+  ## https://www.authelia.com/docs/configuration/identity-providers/oidc.html
+  oidc:
+    ## The hmac_secret is used to sign the JWT's sent by OpenID Connect.
+    ## HMAC Secret can also be set using a secret: https://www.authelia.com/docs/configuration/secrets.html
+    hmac_secret: this_is_a_secret_abc123abc123abc
```

Comment:
maybe we should rename this key because I guess non-technical people will certainly wonder whether the key should be hmac-ed in the config file somehow (even if it does not mean anything). Maybe `signing_secret` would be less techy and more explicit.
What do you think @james-d-elliott ?

Reply:
Yep. Sounds good to me. we can explain the technical side in documentation which is where that's most applicable.

Reply:
My bad this key is used to authenticate the signature of the token, but the signature is generated from the issuer_private_key.

Reply:
Ah yeah right, I'll hold off on the change.

Reply:
I updated the comment.